bin/ip-wrangler-stop

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export __dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+export __dir=$( dirname ${__dir} )
+
+pushd ${__dir}/lib/ 2>&1 >> /dev/null
+    thin stop
```

Comment:
So as far as I understand it only one ipwrangler can be started per user. In the feature app pid file path can be defined in `config.yml` and used while starting/stopping app.